frontend/public/components/nav/admin-nav.tsx

```diff
@@ -78,6 +78,7 @@ const MonitoringNavSection_ = ({ canAccess }) => {
           id="monitoringalerts"
           href="/monitoring/alerts"
           name={t('public~Alerting')}
+          namespaced={false}
```

Comment:
Default value for `namespaced` should be `false` and therefore this change won't be necessary.
```suggestion
```

Reply:
Ack. Solved